tests/test_bookmarks.py

```diff
@@ -271,13 +272,21 @@ def test_run(self):
                     self.assertIn(expected_pk_value, sync_pk_values)
 
                 # Verify updated fields are replicated as expected
-                for updated_record in updated_records[stream]:
-                    expected_updated_key = 'metadata'
-                    expected_updated_value_substring = 'bob'
-                    updated_pk_value = updated_record.get('id')
-                    sync_records_metadata = [sync_record.get('metadata')
-                                             for sync_record in second_sync_data
-                                             if sync_record.get('id') == updated_pk_value]
-                    self.assertTrue(len(sync_records_metadata) == 1)
-                    self.assertIn(expected_updated_value_substring,
-                                  sync_records_metadata[0].get('test_value'))
+                if stream == "payment_intents":
```

Comment:
Add code comment, why we need to add this if-else.

Reply:
Added comments